addons/website_slides/models/gamification_karma_tracking.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class KarmaTracking(models.Model):
+    _inherit = 'gamification.karma.tracking'
+
+    def _get_selection_origin(self):
+        return super()._get_selection_origin() + [('slide.slide', 'Quiz'), ('slide.channel', self.env['slide.channel']._description)]
```

Comment:
Why do you do `self.env['slide.channel']._description` to get the label of the channel, but you just do `Quiz` for the quiz ?